examples/async-examples.ts

```diff
@@ -0,0 +1,47 @@
+/* Initialize your client with your revai access token */
+const token = <YOUR-ACCESS-TOKEN>;
+var client = new RevAiApiClient(token);
+
+/* Get account details */
+var account = await client.getAccount();
+console.log(`Account: {account.email}`);
+console.log(`Balance: {account.balance_seconds} seconds`);
+
+/* Submit job and receive job object */
+var job = await client.submitJobUrl("https://www.rev.ai/FTC_Sample_1.mp3");
+// var job = client.submitJobLocalFile("./path/to/file.mp4");
+
+console.log(`Job Id: {job.id}`);
+console.log(`Status: {job.status}`);
+console.log(`Created On: {job.created_on}`);
+
+/* Check the status of job */
+var jobDetails = await client.getJobDetails(job.id);
+
+/* Get all transcription jobs */
+var listOfJobs = await client.getListOfJobs();
+
+/* Get at most # of jobs */
+var listOfJobs = await client.getListOfJobs(5);
+
+/* Get all jobs started after a certain job id */
+var listOfJobs = await client.getListOfJobs(undefined, job.id);
```

Comment:
I think you can just call this once with all the possible arguments and have a comment saying what the defaults are